src/uct/ib/ud/base/ud_ep.h

```diff
@@ -212,6 +211,7 @@ struct uct_ud_ep {
          uct_ud_psn_t           psn;          /* Next PSN to send */
          uct_ud_psn_t           max_psn;      /* Largest PSN that can be sent */
          uct_ud_psn_t           acked_psn;    /* last psn that was acked by remote side */
+         uct_ud_psn_t           err_skb_count;/* number of failed SKBs on the ep */
```

Comment:
cwnd is used in psn arithmetics ex: ep->tx.max_psn     = ep->tx.psn + ep->ca.cwnd; while err_skb_count is just a counter